### PR TITLE
add precourse

### PR DESCRIPTION
**Pull request by MiguiHidalgo**

### CODE REVIEW

File: staff/miguel-angel-sevillano/precourse/Airlines/Index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  
+  flights.forEach(function(obj){
+  if(obj.id > 5 ){
+    lastFlights.push(obj.to);
+  }
+})
+lastFlights = lastFlights.join(" \n ");
+}
+
+function yesScales(){// que vuelos tienen escala
```
Comment (carlosq09) [inline staff/miguel-angel-sevillano/precourse/Airlines/Index.js:54]: @MiguiHidalgo  Fix identation on `function yesScales()`

File: staff/miguel-angel-sevillano/precourse/Bingo/Index.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    }
+    for (prop in carton.fila3){
+        fila3Show.push(carton.fila3[prop])
+    }
+    fila1Show =fila1Show.join('  ')
+    fila2Show = fila2Show.join(' ')
+    fila3Show = fila3Show.join(' ')
+}
+
+function noBingo(){//nos sirve para confirmar que nosotros no hemos tachado ningun numero y por tanto otro judor si lo habra hecho
```
Comment (carlosq09) [inline staff/miguel-angel-sevillano/precourse/Bingo/Index.js:69]: @MiguiHidalgo Fix identation on function `function noBingo()`

**Review by carlosq09 — CHANGES_REQUESTED** [review]

File: staff/miguel-angel-sevillano/precourse/Airlines/Index.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+      { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+      { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false }
+  ];
+  var avPrice=0;
+  var flightScale =[];
+  var lastFlights =[];
+  var dayFlights =[];
+
+Welcome();
```
Comment (maz-e) [inline staff/miguel-angel-sevillano/precourse/Airlines/Index.js]: @MiguiHidalgo Hi tocayo! I'm not sure but is not better declare the function before call it ? 
Ej:
```javascript
function welcome(){
  //your code
}
welcome();
```
same in line 29, 30, 31, 32 & 33

File: staff/miguel-angel-sevillano/precourse/Airlines/Index.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+avCost();
+lastDestinations();
+yesScales();
+flightsOfDay();
+userAlert();
+  
+function Welcome(){//mensaje de bienvenida
```
Comment (maz-e) [inline staff/miguel-angel-sevillano/precourse/Airlines/Index.js]: better in lowercase
```javascript 
function welcome() {}
```

File: staff/miguel-angel-sevillano/precourse/Airlines/Index.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  }
+})
+lastFlights = lastFlights.join(" \n ");
+}
+
+function yesScales(){// que vuelos tienen escala
+
+  flights.forEach(function(obj){
+       
+       if(obj.scale === true){
```
Comment (maz-e) [inline staff/miguel-angel-sevillano/precourse/Airlines/Index.js]: Hi again! better this, isn't it?
```javascript
if(obj.scale) {
   //your code
} else {
  //your code
}
```

File: staff/miguel-angel-sevillano/precourse/Bingo/Index.js [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+    
+    if (l1ck === false && l2ck === false && l3ck === false){
```
Comment (maz-e) [inline staff/miguel-angel-sevillano/precourse/Bingo/Index.js]: I think that you can improve your code with negation
```javascript
if(!l1ck && !.....)
```

File: staff/miguel-angel-sevillano/precourse/Bingo/Index.js [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+    if(linea3Count === 5 && linea === false){
+        linea = true;
+    alert('Felicidades tienes una linea!!')
+    }
+}
+if(bingo === 15){
+
+    alert('Felicidades! has echo BINGO en: '+turn+' turnos!')
+    next = confirm('Quieres volver a jugar?')
+        if(next === true){
```
Comment (maz-e) [inline staff/miguel-angel-sevillano/precourse/Bingo/Index.js]: same problem like 'Airlines'
```javascript 
if(next) {
    //your code
} else {
   //your code
}
```

File: staff/miguel-angel-sevillano/precourse/Bingo/Index.js [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+        alert ('Esperamos volerte a ver pronto!')
+        end = true;
+    }
+}
+noBingo()
+if(nBingo === 15){
+    nBingo1= true
+    alert('Lo sentimos pero otro jugador a cantado Bingo')
+    next = confirm('Quieres volver a jugar?')
+if(next === true){
```
Comment (maz-e) [inline staff/miguel-angel-sevillano/precourse/Bingo/Index.js]: the same again (like in line 147)

same problems in lines 169 (!), 177 & 180

**Pull request merged**